systests/microprofile/client/weld/testng.xml

```diff
@@ -4,11 +4,11 @@
     <test name="All TCK Tests">
         <packages>
             <package name="org.eclipse.microprofile.rest.client.tck" />
-            <package name="org.eclipse.microprofile.rest.client.tck.asynctests" />
             <package name="org.eclipse.microprofile.rest.client.tck.cditests" />
-            <package name="org.eclipse.microprofile.rest.client.tck.sse" />
+            <package name="org.eclipse.microprofile.rest.client.tck.asynctests" />
```

Comment:
@andymc12 something for us to look at, previous PR builds have test case failures:

```
org.eclipse.microprofile.rest.client.tck.cditests.HasConversationScopeTest.testHasConversationScoped
org.eclipse.microprofile.rest.client.tck.cditests.HasConversationScopeTest.testHasConversationScopedFromConfigKey
```

When run locally, I was able to see it once or twice out of 10 runs. Changing the order of tests seems to cure the problem but for sure the order should not matter, something fishy here.